cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -253,3 +264,21 @@ func addCIFSNonInteractive(manager *hostFolderConfig.Manager, name string) {
 	hostFolder := hostFolderConfig.NewCifsHostFolder(config)
 	manager.Add(hostFolder, !instanceOnly)
 }
+
+// returns the name and full-name for shareType
+func readNameAndTypeInteractive() (string, string) {
+	name := util.ReadInputFromStdin("Name")
+	if name == "" {
+		atexit.ExitWithMessage(1, noName)
+	}
+	shareType := util.ReadInputFromStdin("Type [cifs, sshfs (C/s)]")
+	if shareType == "s" {
+		return name, hostFolderConfig.SSHFS.String()
+	}
+
+	if shareType == "c" || shareType == "" {
```

Comment:
@anjannath , I think you need to also check capital `C`, with following instruction
```
Type [cifs, sshfs (C/s)]: 
```
I would mostly type `C` not `c`,

Reply:
@anjannath best to always do tolower and just verified against `c` and also do same with the sshfs case.

Reply:
As I said, I tried and I got this
```
$ minishift hostfolder add -i
Name: share
Type [cifs, sshfs (C/s)]: C
'C' is an unknown host folder type
```

Reply:
Probably good to check for capital S as well.

Reply:
@budhrg Yes.. thanks will fix this..